ansible/templates/google_symptoms-params-prod.json.j2

```diff
@@ -0,0 +1,7 @@
+{
+  "export_start_date": "2020-02-20",
+  "static_file_dir": "./static",
+  "export_dir": "./common/covidcast/receiving/safegraph_patterns",
```

Comment:
```suggestion
  "export_dir": "./common/covidcast/receiving/google_symptoms",
```

Reply:
Oh! Should be `"/common/covidcast/receiving/google_symptoms"` (sans `.`)?

(Tried to un-resolve this so I could suggest, but I think I'm doing it wrong.

Reply:
@krivard I wasn't sure how to properly update this change. See if it makes sense to you.